src/components/TeamGeneratorBeta.tsx

```diff
@@ -40,20 +40,40 @@ const useStyles = makeStyles({
   dropdownButton:{width:'100%'}
 });
 
+type TeamStats = {
+  color: string,
+  summoners: string[],
+  winRate: number
+}
+
+export const TeamTables = ({ color, summoners, winRate}: TeamStats)=>
```

Comment:
nit: Can we call this TeamTable instead of TeamTables

Reply:
yeah that makes more sense